guides/common/modules/proc_exporting-a-content-view-version-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,75 @@
+[id="Exporting_a_Content_View_Version_in_a_Syncable_Format_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to a syncable format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content through a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead, on the importing {ProjectServer} you must:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set an SSL/TLS CA Credential if the web server requires it.
+* Enable the repository.
+* Synchronize the repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization using content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+
+For more information, see xref:Importing_a_Content_View_Version_{context}[].
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+
+* File repositories
+* Kickstart repositories
+* RPM repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories.
+endif::[]
+
+The export contains the following data:
+
+* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```

Comment:
```suggestion
* Directories containing the RPMs, *listing* files and metadata of each repository in yum format that can be used to synchronize in the importing {ProjectServer}.
```

Would it make sense to reword "RPMs" to "packages"? Or even something more generic that also applies to file repositories?